app/src/ui/branches/pull-request-list-item.tsx

```diff
@@ -68,8 +90,15 @@ export class PullRequestListItem extends React.Component<
     })
 
     return (
-      <div className={className}>
-        <Octicon className="icon" symbol={OcticonSymbol.gitPullRequest} />
+      <div
+        className={className}
+        onMouseEnter={this.onMouseEnter}
+        onMouseLeave={this.onMouseLeave}
+        onMouseUp={this.onMouseUp}
+      >
+        <div>
```

Comment:
Changed to be encased in a div so that flex styling would center it vertically when I made the .pull-request-item class take 100%.